### PR TITLE
[BPK-2158] Add screenshots and missing Switch story

### PR DESCRIPTION
**Pull request by shaundon**

### CODE REVIEW

File: docs/Icon/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Exampe usage in XML
+
+```
+<ImageView
+android:layout_width="16dp"
+android:tint="@color/bpkGray900"
+android:layout_margin="@dimen/bpkSpacingSm"
+app:srcCompat="@drawable/bpk_flight"
+android:layout_height="16dp"/>
```
Comment (matthewdavidson) [inline docs/Icon/README.md]:
```suggestion
  android:layout_height="16dp" />
```

File: docs/Icon/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+They can be used in code and XML.
+
+Exampe usage in XML
+
+```
+<ImageView
+android:layout_width="16dp"
+android:tint="@color/bpkGray900"
+android:layout_margin="@dimen/bpkSpacingSm"
+app:srcCompat="@drawable/bpk_flight"
```
Comment (matthewdavidson) [inline docs/Icon/README.md]:
```suggestion
  app:srcCompat="@drawable/bpk_flight"
```

File: docs/Icon/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+They can be used in code and XML.
+
+Exampe usage in XML
+
+```
+<ImageView
+android:layout_width="16dp"
+android:tint="@color/bpkGray900"
+android:layout_margin="@dimen/bpkSpacingSm"
```
Comment (matthewdavidson) [inline docs/Icon/README.md]:
```suggestion
  android:layout_margin="@dimen/bpkSpacingSm"
```

File: docs/Icon/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Icons are converted to image vectors and are available as drawables.
+
+They can be used in code and XML.
+
+Exampe usage in XML
+
+```
+<ImageView
+android:layout_width="16dp"
+android:tint="@color/bpkGray900"
```
Comment (matthewdavidson) [inline docs/Icon/README.md]:
```suggestion
  android:tint="@color/bpkGray900"
```

File: docs/Icon/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Icons are converted to image vectors and are available as drawables.
+
+They can be used in code and XML.
+
+Exampe usage in XML
+
+```
+<ImageView
+android:layout_width="16dp"
```
Comment (matthewdavidson) [inline docs/Icon/README.md]:
```suggestion
  android:layout_width="16dp"
```

**Comment by shaundon** [comment]
@matthewdavidson I think you're right about the panel, I'm investigating it now.

**Pull request merged**